lib/event_store/sql/statements/insert_events.sql.eex

```diff
@@ -24,12 +24,17 @@ WITH
     <% end %>
   ),
   stream AS (
-    <%= if stream_id do %>
+    <%= cond do %>
```

Comment:
This makes stream_id and created_at mutually exclusive. Not sure that that's a good idea?

Reply:
I couldn't think of a use case for both updating `stream_version` and the `created_at`. Do you think there is a reasonable use case for this? I am happy to update this file if you think we should support this.